components/admin_console/system_users/system_users.jsx

```diff
@@ -304,7 +304,7 @@ export default class SystemUsers extends React.Component {
                         value={this.props.filter}
                         onChange={this.handleFilterChange}
                     >
-                        <option value=''>{'None'}</option>
+                        <option value={SearchUserTeamFilter.ALL_USERS}>{Utils.localizeMessage('admin.system_users.allUsers', 'All Users')}</option>
```

Comment:
The UI now looks good, but is the value for this option supposed to change? It seems like it should probably still be the empty string
```suggestion
                        <option value=''>{Utils.localizeMessage('admin.system_users.allUsers', 'All Users')}</option>
```